source/val/validate_builtins.cpp

```diff
@@ -667,6 +667,36 @@ class BuiltInsValidator {
   // instruction.
   void Update(const Instruction& inst);
 
+  uint32_t GetMeshEntryPoint() {
+    if (mesh_entry_point_ == 0) {
+      for (const uint32_t entry_point : _.entry_points()) {
+        // Every entry point from which this function is called needs to have
+        // Execution Mode DepthReplacing.
+        const auto* models = _.GetExecutionModels(entry_point);
+        if (models->find(spv::ExecutionModel::MeshEXT ) != models->end() ||
```

Comment:
the CI bot has not kicked off yet, but it will for sure fail for not running clang-format, so I would suggest running it on this PR

```suggestion
        if (models->find(spv::ExecutionModel::MeshEXT) != models->end() ||
```